Exiled.API/Features/Player.cs

```diff
@@ -191,6 +192,7 @@ private set
                 CameraTransform = value.PlayerCameraReference;
 
                 value.playerStats._dictionarizedTypes[typeof(HealthStat)] = value.playerStats.StatModules[0] = healthStat = new CustomHealthStat { Hub = value };
+                value.playerStats._dictionarizedTypes[typeof(HumeShieldStat)] = value.playerStats.StatModules[0] = humeshieldStat = new CustomHumeShieldStat { Hub = value };
```

Comment:
```suggestion
                value.playerStats._dictionarizedTypes[typeof(HumeShieldStat)] = value.playerStats.StatModules[4] = humeshieldStat = new CustomHumeShieldStat { Hub = value };
```
Stat 4 is the hume shield stat. Stat 0 is the health stat.